Dockerfile

```diff
@@ -1,5 +1,7 @@
 FROM node:9-slim
 
+RUN apt-get update && apt-get install -y git-core
+
```

Comment:
As we talked about over slack, `RUN yarn config set "strict-ssl" false` should be removed below, perhaps replaced with the commented out block below that can be un-commented for local testing, else that block and the update needed to test locally should be added to the docker documentation on gitlab.

```
# Add MITRE Soft Certs
# ADD http://pki.mitre.org/MITRE%20BA%20Root.crt /usr/local/share/ca-certificates/
# ADD http://pki.mitre.org/MITRE%20BA%20NPE%20CA-1.crt /usr/local/share/ca-certificates/
# ADD http://pki.mitre.org/MITRE%20BA%20NPE%20CA-3.crt /usr/local/share/ca-certificates/
# RUN cat /usr/local/share/ca-certificates/MITRE*.crt >> /usr/local/share/ca-certificates/MITRE_Cert_Chain.txt
# RUN update-ca-certificates
# RUN yarn config set cafile /usr/local/share/ca-certificates/MITRE_Cert_Chain.txt
```